translate/src/modules/editor/hooks/useSendTranslation.ts

```diff
@@ -87,9 +87,9 @@ export function useSendTranslation(): (ignoreWarnings?: boolean) => void {
       );
 
       const badgeLevel = content.badge_update?.level;
-      if (badgeLevel) {
+      if (content.badge_update?.name && badgeLevel) {
```

Comment:
Since this pattern is used twice, could you please instead declare it above?

`const badgeName = content.badge_update?.name;`,